docs/examples/and.rst

```diff
@@ -194,62 +195,87 @@ following coefficients:
-
->>> print(sampler.adjacency[sampler.nodelist[0]])      # doctest: +SKIP
-{128, 4, 5, 6, 7}
-
-For the D-Wave 2000Q system the above code ran on, you see that the first available qubit
-is adjacent to qubit 4 and four others. On an Advantage system with its Pegasus
-topology, you might see an output such as this:
+The next line of code looks at the ``nodelist`` property of the sampler. Select 
+the first node, which on a QPU is a qubit, and print its adjacent nodes, which 
+are the qubits it is coupled to.
 
+>>> sampler.nodelist[0]   # doctest: +SKIP
+30
 >>> print(sampler.adjacency[sampler.nodelist[0]])      # doctest: +SKIP
-{31, 2940, 2955, 2970, 2985}
-
-You can map the NOT problem's two linear coefficients and single quadratic coefficient,
-:math:`q_1=q_2=-1` and :math:`q_{1,2}=2`, to biases on the D-Wave 2000Q's qubits 0 and 4
-and coupling (0, 4). The figure below shows a minor embedding of the NOT gate into
-the D-Wave 2000Q QPU unit cell (four horizontal qubits connected to four vertical qubits
-via couplers).
-
-.. figure:: ../_images/Embedding_Chimera_NOT.png
-   :name: Embedding_Chimera_NOT
-   :alt: image
-   :align: center
-   :scale: 90 %
-
-   A NOT gate minor embedded into the topmost left unit cell of a
-   D-Wave 2000Q QPU. Variables :math:`x_1,x_2` are minor
-   embedded as qubits 0 and 4 (blue circles). Biases :math:`q_1,q_2=-1,-1`
-   and coupling strength :math:`q_{1,2}=2` are also shown.
-
-The following code uses the :class:`~dwave.system.composites.FixedEmbeddingComposite` composite
-to manually minor-embed the problem in the D-Wave 2000Q QPU. Its last line prints a confirmation
-that indeed the two selected qubits are adjacent (coupled).
+{2985, 2955, 45, 2970, 2940, 31}
+
+For the Advantage system the above code ran on in this particular execution, 
+you see that the first available qubit, ``30``, is adjacent to qubit ``31`` 
+and a few others. 
+
+You can map the NOT problem's two linear coefficients (coefficients of the 
+:math:`-1x` and :math:`-1z` terms) and single quadratic coefficient (coefficient 
+of the :math:`2xz` term) to :math:`q_1=q_2=-1` biases applied to the  Advantage's 
+qubits ``30`` and ``31`` and a :math:`q_{1,2}=2` coupling strength applied to 
+coupler ``[30, 31]``. The figure below shows this embedding of the NOT gate 
+onto an Advantage QPU.
+
+.. figure:: ../_images/embedding_pegasus_NOT.png
+    :name: Embedding_Pegasus_NOT
+    :alt: image
+    :align: center
+    :scale: 50 %
+
+    A NOT gate minor embedded onto an Advantage QPU. Variables :math:`x,z` 
+    (left) are minor-embedded to qubits ``30`` and ``31`` (right). This 
+    visualization is produced by Ocean's :std:doc:`docs_inspector/sdk_index` tool.
+    
+Notice that coupling strength is :code:`BIAS 0.5` above: as explained in the 
+:std:doc:`system documentation <sysdocs_gettingstarted:doc_getting_started>`,
+the QPU can be viewed as minimizing the Ising model of [1_] above, with linear 
```

Comment:
autonumbered footnot can be used here as well

Reply:
Required some modification of the text and your naming but looks nice now: 
![image](https://user-images.githubusercontent.com/34041130/234375811-15d62854-919b-421d-ac2d-5077afa04747.png)
![image](https://user-images.githubusercontent.com/34041130/234375929-7efc0c60-4409-4180-bcaf-37c69e459d22.png)